### PR TITLE
asm.cmt.off: Accept 0 & 1 as aliases for false & true respectively

### PR DESCRIPTION
**Pull request by kazarmy**

### CODE REVIEW

File: libr/core/cconfig.c [hunk 1]
```diff
@@ -2254,6 +2254,14 @@ static bool cb_seggrn(void *user, void *data) {
 	return true;
 }
 
+static bool cb_cmtoff(void *user, void *data) {
+	RConfigNode *node = (RConfigNode *) data;
+	if (node->i_value || !strcmp (node->value, "0")) {
+		free (node->value);
+		node->value = strdup (node->i_value ? "true" : "false");
```
Comment (radare) [inline libr/core/cconfig.c]: Use rstr api for that

**Comment by radare** [comment]
This should be generic across all the config vars. Not specific to this one

File: libr/core/cconfig.c (hunk 1 above)
Comment (kazarmy) [reply libr/core/cconfig.c]: ok dce54f8af9f4c52d48366074ecf2d8b0c9dbf2ef

File: libr/util/str.c [hunk 2]
```diff
@@ -3467,12 +3467,19 @@ R_API const char *r_str_bool(int b) {
...
 
+R_API bool r_str_is_false(const char *s) {
+	return !r_str_casecmp ("no", s)
+		|| !r_str_casecmp ("off", s)
+		|| !r_str_casecmp ("false", s)
+		|| !r_str_casecmp ("0", s);
+}
+
 R_API bool r_str_is_bool(const char *val) {
 	if (!r_str_casecmp (val, "true") || !r_str_casecmp (val, "false")) {
```
Comment (radare) [inline libr/util/str.c]: why is_bool is not using is_true || is_false ? :D

Comment (kazarmy) [reply libr/util/str.c]: oh ok 430d0f91b76473336c24a00d48f8d28cf8854785 didn't realize that :D

**Comment by kazarmy** [comment]
ok to merge?

**Pull request merged**